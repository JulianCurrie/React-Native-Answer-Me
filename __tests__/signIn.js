import 'react-native';
import React from 'react';
import SignIn from '../src/components/auth/signIn';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <SignIn />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
