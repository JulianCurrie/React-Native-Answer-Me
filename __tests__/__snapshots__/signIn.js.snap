exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "azure",
      "flex": 1,
      "justifyContent": "center",
      "padding": 40,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontFamily": "Elegante-Regular",
        "fontSize": 50,
      }
    }>
    Answer Me
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "red",
        "textAlign": "center",
      }
    }>
    
  </Text>
  <TextInput
    onChangeText={[Function]}
    placeholder="Email"
    style={
      Object {
        "alignSelf": "center",
        "borderColor": "black",
        "borderRadius": 5,
        "borderWidth": 1,
        "height": 50,
        "marginTop": 10,
        "textAlign": "center",
        "width": 250,
      }
    }
    underlineColorAndroid="transparent" />
  <TextInput
    onChangeText={[Function]}
    placeholder="Password"
    secureTextEntry={true}
    style={
      Object {
        "alignSelf": "center",
        "borderColor": "black",
        "borderRadius": 5,
        "borderWidth": 1,
        "height": 50,
        "marginTop": 10,
        "textAlign": "center",
        "width": 250,
      }
    }
    underlineColorAndroid="transparent" />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderColor": "black",
        "borderRadius": 5,
        "borderWidth": 1,
        "height": 50,
        "justifyContent": "center",
        "marginTop": 10,
        "opacity": 1,
        "width": 250,
      }
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "textAlign": "center",
        }
      }>
      Sign In
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "justifyContent": "space-between",
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "blue",
            "marginTop": 10,
          }
        }>
        Sign Up
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "blue",
            "marginTop": 10,
          }
        }>
        Forgot Password
      </Text>
    </View>
  </View>
</View>
`;
